### ISSUE TITLE
[MHV Portal / Accessibility] Selecting current page link in breadcrumb moves focus to MHV nav and does not display focus indicator

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#123109]
### 📝 User story

As a keyboard user, I want to select the current page link in the breadcrumbs and have visible focus set to the page heading (as it does on other breadcrumb links), so that I can predict what will happen when I click the current page link and I know where focus is when the current page loads.

### 📝 Problem statement

Selecting the current page link in the breadcrumbs reloads the current page, sets focus to the start of the MHV secondary nav, and does not display a focus indicator.

### 👣 Reproduction steps

1. Use the Tab key to navigate to the Medical Records > Care summaries and notes list page.
2. Notice the focus indicator on the Care summaries and notes heading.
3. Use the Tab key to navigate back to the breadcrumbs menu.
4. Select the "Care summaries and notes" page in the breadcrumb nav.
5. Notice that the page reloads and the focus indicator is not visible.
6. Press the Tab key.
7. Notice the focus indicator appears on the My HealtheVet link in the secondary nav.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility standards
- [Guideline 2.4 Navigable: Provide ways to help users navigate, find content, and determine where they are](https://www.w3.org/WAI/WCAG21/Understanding/navigable)
  - [SC 2.4.3 Focus Order (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

#### Experience Standards
- [Consistency: User has a predictable and unchanging experience across all VA.gov and VA Mobile App services, tools, and pages](https://design.va.gov/about/experience-standards/#category-consistency)
  - [User experiences predictable interactions with components and patterns](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components-and-patterns)

### 𝌣 Proposed solution or next steps

Set the focus to the page heading after activating the current page link.

### ✅ Acceptance criteria

- [x] The system displays a focus indicator on the page heading when the page loads when accessed using the current page link in the breadcrumbs.
- [x] The user can see the focus indicator on the page heading after activating the current page link.
- [x] The user can use the keyboard to navigate to the content that appears after the page heading in the code order.

### ✅ Possible tasks

- [x] Review, refine, and acknowledge feedback.
- [x] Update ticket information (title, assignee, labels, project(s), etc.)
- [x] Fix and/or document decisions made.
- [x] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [x] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by sarahhortonadhoc** [comment]
Actioning this issue would involve changing the behavior of the current page link in the breadcrumb nav, so that selecting the current page link would reload the current page with focus set to the heading 1, which is the [default behavior focus management pattern within MHV](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/design/documentation/decisionrecords.md#focus-management-on-page-load). 

It would also be an opportunity to update the [breadcrumb visual design to match the VADS design as specified in Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=798-931). (See [Slack discussion about current page link in Breadcrumbs component](https://dsva.slack.com/archives/C01DBGX4P45/p1759334564211189) for more insight into the design specs.)

@fmccaf1 I can get started on this work if it would be helpful, as part of the [accessibility team's current sprint](https://github.com/orgs/department-of-veterans-affairs/projects/1748/views/17). Let me know!

**Comment by steveg-IAT** [comment]
@sarahhortonadhoc 
Don't want to push this late on Friday.  Let's review from my local during Tues sync. I can push afterwards.

**Comment by steveg-IAT** [comment]
[PR #41545](https://github.com/department-of-veterans-affairs/vets-website/pull/41545) IN:
- [ ] Tests Past
- [ ] Reviewed
- [ ] Merged
